examples/ord-min-max.js

```diff
@@ -0,0 +1,42 @@
+const {Max, Min, Ord} = require('../fantasy-monoids');
+const log = (x) => console.log(x);
+const concat = (x, y) => x.concat(y);
+const mconcat = (xs, empty) => xs.length ? xs.reduce(concat) : empty();
```

Comment:
@SimonRichardson `mconcat` would be a great helper. What say?
Should something like this go to `fantasy-helpers` or stay with the monoids?